ci/common.sh

```diff
@@ -0,0 +1,12 @@
+# Common material sourced by CI scripts in this directory
+
+# On Cirrus, oversubscribe the CPUs when on Linux. This uses Cirrus' "greedy" feature.
+if [ "${CIRRUS_OS}" = linux ]; then
```

Comment:
In nitpicky comments. If this is using bash (which it might not) - a lot of people say that you should prefer `[[ ... ]]` over `[ ... ]` nowadays. If this is indeed a bash script that might be worth changing.

Reply:
The two scripts (test.sh and build.sh) calling this are indeed Bash. They're already a bit inconsistent in `[ .. ]` vs `[[ .. ]]`, `=` vs `==`, all the usual stuff. Let me make a quick edit pass to bashify. Seems pretty unlikely that we need these particular scripts to be portable shell.